src/Tasks/Microsoft.Common.CurrentVersion.targets

```
@@ -2217,7 +2223,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     ============================================================
     -->
   <PropertyGroup>
-    <GetTargetPathWithTargetPlatformMonikerDependsOn>$(GetTargetPathDependsOn)</GetTargetPathWithTargetPlatformMonikerDependsOn>
+    <GetTargetPathWithTargetPlatformMonikerDependsOn>$(GetTargetPathWithTargetPlatformMonikerDependsOn);$(GetTargetPathDependsOn)</GetTargetPathWithTargetPlatformMonikerDependsOn>
```

I think I'd swap this one.

```suggestion
    <GetTargetPathWithTargetPlatformMonikerDependsOn>$(GetTargetPathDependsOn);$(GetTargetPathWithTargetPlatformMonikerDependsOn)</GetTargetPathWithTargetPlatformMonikerDependsOn>
```